api/app/services/feedback_service.py

```
@@ -738,6 +744,345 @@ def get_source_weights(self) -> Dict[str, float]:
+                if date_from:
+                    from_date = datetime.fromisoformat(date_from)
+                    if item_date < from_date:
+                        continue
+
+                if date_to:
+                    to_date = datetime.fromisoformat(date_to)
+                    if item_date > to_date:
+                        continue
+
+                filtered.append(item)
+            except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)